tests/user_commands/tests.py

```diff
@@ -119,7 +119,7 @@ def test_call_command_option_parsing_non_string_arg(self):
         It should be possible to pass non-string arguments to call_command.
         """
         out = StringIO()
-        management.call_command('dance', 1, verbosity=0, stdout=out)
+        management.call_command('dance', 1, stdout=out)
```

Comment:
Only this parameter is unnecessary, rest of them is necessary to hide std outputs.

Reply:
Here we display a message even with `verbosity=0`, so I think it's worth to leave it.